pkg/operation/cloudbotanist/gcpbotanist/infrastructure.go

```diff
@@ -155,6 +161,45 @@ func (b *GCPBotanist) destroyKubernetesFirewallRules(ctx context.Context) error
 	return nil
 }
 
+func (b *GCPBotanist) destroyKubernetesRoutes(ctx context.Context) error {
+	t, err := b.NewShootTerraformer(common.TerraformerPurposeInfra)
```

Comment:
This code is now duplicated. Can you please move it into a separate function and call it here and in the other one (where you delete firewall routes)?